### DISCUSSION TITLE
Starting Thai Translation for Vue 3 Docs 🇹🇭

### DISCUSSION THREAD

**Discussion by runyasak** [vuejs/docs#2977] [Show and tell]
Hi Thai Vue folks.

I’m starting to translate the Vue 3 documentation into Thai. If you’d like to contribute or have any feedback, please let me know in this discussion.

Repo: https://github.com/runyasak/docs-th

----

สวัสดีครับพี่น้อง Vue ชาวไทย

ผมกำลังจะแปล Documentation ของ Vue 3 เป็นภาษาไทยครับ
ถ้าพี่ ๆ คนไหนอยากช่วย contribute หรือมีข้อคิดเห็นใด ๆ เพิ่มเติม สามารถบอกใน discussion นี้ได้เลยครับ

Repo: https://github.com/runyasak/docs-th

**Comment by Jinjiang** [comment]
@runyasak good job!
Feel free to refer to our translation guidelines here: https://github.com/vuejs-translations/guidelines
and you are also welcome to open a discussion to let more people know your plan: https://github.com/orgs/vuejs-translations/discussions
Thanks.

**Reply by runyasak** [reply]
Hi @Jinjiang. Happy to see you here. 😂
I will follow the guidelines as much as I can.
If I miss any point or do you have any feedback, please let me know.

**Comment by runyasak** [comment]
My current plan is to translate every page in `/src`.
I will create issues for each page or task to let anyone who wants to contribute.

My plan is so simple right now. 😂
